R/reportEDGETransport.R

```diff
@@ -288,15 +289,118 @@ reportEDGETransport <- function(output_folder=".",
                     "EJ/yr", "V1", "det_veh")))
 
     }
+
+    df <- rbind(
+      datatable[!is.na(aggr_mode), sum(demand_EJ, na.rm = T),
+        by = c("region", "year", "aggr_mode", "remind_rep")
+      ][, variable := paste0(aggr_mode, "|", remind_rep)][,c("region", "year", "variable", "V1")],
+      datatable[!is.na(aggr_veh), sum(demand_EJ, na.rm = T),
+        by = c("region", "year", "aggr_veh", "remind_rep")
+      ][, variable := paste0(aggr_veh, "|", remind_rep)][,c("region", "year", "variable", "V1")]
+    )
+    
+    # splits Liquids variables into Biomass, Fossil and Hydrogen according to FE demand shares
+    .split_liquids <- function(df) {
+      demFeSector <- readGDX(gdx, "vm_demFeSector", field = "l", restore_zeros = F)
+
+      # biomass share in transport sector
```

Comment:
I would write biomass share in biomass+hydrogen liquids in total transport sector

and same below... 

So, this only applies same shares of biomass within hydrogen+biomass per sector not within biomass+hydrogen+fossil. The latter can differ in runs with different carbon prices on different transport sectors.

Reply:
thanks, makes sense. i just updated the comments